common/catalog/docker/docker.bom

```diff
@@ -271,14 +269,12 @@ brooklyn.catalog:
           brooklyn.config:
             uniqueTag: docker-bind-url-generator
             enricher.suppressDuplicates: false
-            enricher.triggerSensors:
-              - $brooklyn:sensor("docker.port")
```

Comment:
This isn't going to work any more, as there are no incoming sensors. We will need something like a `StaticSensor` initializer instead.